atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,58 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+
+	protected ListaEncadeada l;
+
+	public PilhaComLista() {
+		this.l = new ListaEncadeada();
+	}
+
+	@Override
+	public void push(Integer element) throws Exception {
+		if (isFull()) {
+			throw new Exception("Pilha cheia.");
+		} else {
+			l.insert(element);
+		}
+	}
+
+	@Override
+	public Integer pop() throws Exception {
+		if (isEmpty()) {
+			throw new Exception("Pilha vazia");
+		} else {
+			int removed = l.data;
+			l.remove(removed);
+			return removed;
+		}
+	}
+
+	@Override
+	public Integer top() throws Exception {
+		if (isEmpty()) {
+			throw new Exception("Pilha vazia.");
+		} else {
+			return l.data;
+		}
+	}
+
+	@Override
+	public boolean isEmpty() {
+		if (l.isEmpty()) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	@Override
+	public boolean isFull() {
+		if (l.size() >= 10) {
```

Comment:
Apenas um ponto de atenção, cuidado com magic number ou valores constantes soltos pelo código, o ideal é levar esse valor para uma variável que represente o contexto daquela informação, por exemplo:

primeiro declara a variável constante com um nome que descreve o valor que ela receber
```
private final int MAX_ELEMENT = 10;
```

```suggestion
		if (l.size() >= this.MAX_ELEMENT) {
```